package/yast2-schema-default.spec

```diff
@@ -75,7 +75,7 @@ BuildRequires: yast2-network >= 4.4.29
 BuildRequires: yast2-nfs-client
 BuildRequires: yast2-nfs-server
 # YaST NIS packages are dropped from TW (bsc#1183893)
-%if 0%{?suse_version} > 1500
+%if 0%{?sle_version}
```

Comment:
Do we really expect that to still be true in SLE > 15? Might be better to check for suse_version <= 1500

Reply:
I do not know. The question is what SLE > 15 will be based on. And probably nobody knows that now... We will need to evaluate all `*_version` macros in the future anyway so I think it does not matter much.

Reply:
Well, what we know so far is that ALP is using 100% TW sources